src/utils/MasterFisher.ts

```diff
@@ -70,6 +71,11 @@ export class MasterFisher implements Fishable {
         if (fishable instanceof FSHTank) {
           result.sdType = this.findSdType(result, types, fishables);
         }
+        if (fishable instanceof FSHTank && !result.resourceType) {
+          result.resourceType = this.fishForMetadata(
+            (fishable.fish(item, Type.Instance) as Instance)?.instanceOf
+          )?.sdType;
+        }
```

Comment:
I remember why you're doing it this way because we discussed it yesterday.  The future me (in 6 months) is unlikely to remember that.  Could you add a brief comment regarding what is happening here (and why it is happening)?

Reply:
Yes this is mysterious, 4ba168f adds a demystifying comment.